content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -264,6 +264,102 @@ $ export SOURCE_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata
 $ kubectl exec -it $SOURCE_POD -c sleep curl http://httpbin.org/headers
 {{< /text >}}
 
+## Install Istio with access to all the external services by default
+
+An alternative to calling external services directly is to instruct Istio proxy to pass through the calls to all the
```

Comment:
instruct Istio -> instruct the Istio

Reply:
Fixed in https://github.com/istio/istio.io/pull/3140/commits/8ced2aa8394969812896bf71b3552e7be015c991